galaxy_skeleton2.R

```diff
@@ -78,11 +85,12 @@ variable_names <-
 # replace all "|" with "," and remove all empty space. The gsub-function is
 # super for this kind of search-and-replace. Replace the question mark below.
 
-comma_separated_values <- 
-  ? %>% 
+
+
+
+comma_separated_values <- raw_file[L:length(raw_file)] %>% 
```

Comment:
Using index raw_file[(L+1):length(raw_file)] would have removed the seperator line here. you get a following error because including the seperator line, forces you to save the csv file including this line. Always make sure to complete the formatting of the data, before saving it as a new file.